overlay.d/05rhcos/usr/libexec/coreos-update-bootloader

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+set -euo pipefail
+
+# https://issues.redhat.com/browse/OCPBUGS-54594
+
+# This script updates the bootloader using bootupd when it's safe
+# and manually otherwise. Right now bootupd doesn't support RAID-1
+# setups and also not sure of the behavior if there are multiple
+# EFI-SYSTEM labeled filesystems attached.
+
+# Function that actually does the manual copy
+copy_to_esp_device() {
+    local device=$1
+    mount $device /boot/efi
+    echo "[Before Update: ${device}]"
+    find /boot/efi/ -type f | xargs md5sum
```

Comment:
```suggestion
    find /boot/efi/ -type f | xargs sha256sum
```
I know we are not using it for security here but let's use sha256sum to avoid later compliance complains.

Reply:
hmm. I understand your point but also contend that the extra length in a log is a bit unwieldy (esp on serial console).

How much of a concern do you really think this is? Also of note that (at least for right now) this is only in 4.18, so it's "life" will die with 4.18.

Reply:
ok I added it.